libs/orders/src/lib/components/order-list/order-list.tsx

```diff
@@ -329,7 +329,7 @@ export const isOrderActive = (status: Schema.OrderStatus) => {
 };
 
 export const isOrderAmendable = (order: Order | undefined) => {
-  if (!order || order.peggedOrder || order.liquidityProvision) {
```

Comment:
Why we are removing this condition?

Reply:
because pegged orders are amendable

Reply:
here is the spec https://github.com/vegaprotocol/specs/blob/master/protocol/0024-OSTA-order_status.md#parked-orders